includes/api/class-wc-rest-orders-controller.php

```diff
@@ -121,6 +121,18 @@ public function register_routes() {
 			),
 			'schema' => array( $this, 'get_public_batch_schema' ),
 		) );
+
```

Comment:
This changes should be moved into a new class, creating a new schema too, since includes new items to the current orders schema.